[LTP] Remove flaky futex_wait03 and sigpending02 tests

dimakuv opened the pull request:
These tests were added in commit 6f64f3a but they are flaky. Removing them for now until we debug their bugs and fix them.

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/oscarlab/graphene-tests/11)
<!-- Reviewable:end -->

donporter reviewed (approved):
> Reviewed 1 of 1 files at r1.
> *[Reviewable](https://reviewable.io/reviews/oscarlab/graphene-tests/11)* status: all files reviewed, all discussions resolved, not enough approvals from maintainers (1 more required)



<!-- Sent from Reviewable.io -->

mkow reviewed (approved):
> Reviewed 1 of 1 files at r1.
> *[Reviewable](https://reviewable.io/reviews/oscarlab/graphene-tests/11)* status: :shipit: complete! all files reviewed, all discussions resolved



<!-- Sent from Reviewable.io -->

Commit:
[LTP] Remove flaky futex_wait03 and sigpending02 tests

These tests were added in commit 6f64f3a but they are flaky. Removing
them for now until we debug their bugs and fix them.
